packages/components/src/components/select/select.stories.ts

```diff
@@ -71,600 +57,309 @@ export default {
-        ],
-        args
-      })}
-    </div>`;
-  },
+export const Multiple = {
+  render: () => html`
+    <div class="w-[300px] h-[500px]">
+      <sd-select size="lg" label="Label" placement="bottom" multiple="" value="option-1 option-2">
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
+};
 
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-select');
-    await waitUntil(() => el?.shadowRoot?.querySelector('input'));
-    el?.shadowRoot?.querySelector('input')!.focus();
-  }
+/**
+ * Use the `useTags` attribute to display selected options as tags using the `sd-tag` component.
+ *
+ * **Hint:** it requires the `multiple` attribute to be set.
+ */
+
+export const useTags = {
+  render: () => html`
+    <div class="w-[300px] h-[500px]">
+      <sd-select size="lg" label="Label" placement="bottom" multiple="" value="option-1 option-2" useTags>
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
 };
 
 /**
- * Use `<sl-divider>` to group listbox items visually. You can also use `<small>` to provide labels, but they won’t be announced by most assistive devices.
+ * Use the `max-options-visible` attribute to define the maximum number of selected options that will be visible.
+ *
+ * **Hint:** it requires the `multiple` and `useTags` attributes to be set.<br />After the maximum number of options is reached, the select will display a message indicating how many more options are selected.<br />To remove the limit, set the attribute to `0`.
  */
 
-export const SampleGroupingOptions = {
-  name: 'Sample: Grouping Options',
-  parameters: {
-    controls: {
-      exclude: ['open-attr', 'default']
-    }
-  },
-  render: (args: any) => {
-    return html`<div class="h-[290px] w-[420px]">
-      ${generateTemplate({
-        constants: [
-          {
-            type: 'slot',
-            name: 'default',
-            value:
-              '<div class="text-black px-4 font-bold">Nisi eu excepteur anim esse</div><sd-option value="option-1">Option 1</sd-option><sd-option value="option-2">Option 2</sd-option><sd-divider class="mb-2"></sd-divider><div class="text-black px-4 font-bold">Nisi eu excepteur anim esse</div><sd-option value="option-3">Option 3</sd-option>'
-          }
-        ],
-        args
-      })}
-    </div>`;
-  }
+export const MaxOptionsVisible = {
+  render: () => html`
```

Comment:
Show it like that: (but closed of course) ... Would like to keep the option tags in one row

![image](https://github.com/user-attachments/assets/0d35a263-7a33-4005-8f5d-bf7716ce8016)

Reply:
Updated all selects to be the same size, exceptions are size and placement

Reply:
Updated